lib/Elasticsearch/composer.json

```
@@ -17,12 +17,12 @@
     "require": {
         "php": "^7.1",
         "rollerworks/search": "^2.0@dev,>=2.0.0-ALPHA13",
-        "psr/simple-cache": "^1.0"
+        "psr/simple-cache": "^1.0",
+        "ruflin/elastica": "^3. 2"
```

Extra space between operator.

```suggestion
        "ruflin/elastica": "^3.2"
```